src/main/java/uk/ac/cam/cl/dtg/isaac/api/EventsFacade.java

```diff
@@ -21,17 +21,13 @@
 import com.google.common.collect.Sets;
 import com.google.inject.Inject;
 import com.google.inject.name.Named;
+import com.opencsv.CSVWriter;
 import io.swagger.annotations.Api;
 import io.swagger.annotations.ApiOperation;
 import org.jboss.resteasy.annotations.GZIP;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import uk.ac.cam.cl.dtg.isaac.api.managers.DuplicateBookingException;
-import uk.ac.cam.cl.dtg.isaac.api.managers.EventBookingManager;
-import uk.ac.cam.cl.dtg.isaac.api.managers.EventBookingUpdateException;
-import uk.ac.cam.cl.dtg.isaac.api.managers.EventDeadlineException;
-import uk.ac.cam.cl.dtg.isaac.api.managers.EventIsFullException;
-import uk.ac.cam.cl.dtg.isaac.api.managers.EventIsNotFullException;
+import uk.ac.cam.cl.dtg.isaac.api.managers.*;
```

Comment:
Our CheckStyle suggests against any `import *`'s that aren't constants, so ideally we'd fix these. Once you've updated the IntelliJ config, then "Optimise Imports" should do the right thing.